src/ui/common_components/ExpressionOptions.tsx

```diff
@@ -111,12 +112,37 @@ class ExpressionOptions extends React.Component<ExpressionOptionsProps> {
         (cookie) => cookie.domain === '' && regExp.test(cookie.path),
       );
     } else {
-      cookies = await browser.cookies.getAll(
-        returnOptionalCookieAPIAttributes(this.props.state, {
-          domain: `${trimDotAndStar(exp)}${exp.endsWith('.') ? '.' : ''}`,
-          storeId: this.toPublicStoreId(expression.storeId),
-        }),
-      );
+      try {
```

Comment:
These nested try catch blocks looks complicated, can they be broken up?

Reply:
I think I resolved this through 9c92dde.  Logically this should work.